release_ios.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
 
+options=':t'
+while getopts $options option; do
+  case "$option" in
+    t) TAG=$OPTARG;;
+  esac
+done
```

Comment:
I don't know why but when I pass the -t argument in the scripts it's always an empty string.
For example if I run /.release_ios.sh -t 0.0.1 I get the list of tags since the git tag command is called with an empty string instead of creating one